packages/deadgods/src/utils/index.ts

```diff
@@ -12,9 +12,11 @@ import { IDL as IDL_GEM_FARM, GemFarm } from "./idl-gem-farm";
 //
 // On connection to the host environment, warm the cache.
 //
-ReactXnft.events.on("connect", () => {
-  fetchDegodTokens(window.xnft.publicKey, window.xnft.connection);
-});
+window.addEventListener("load", () => {
```

Comment:
Had to move from `ReactXnft.events` to `window.xnft.solana.addListener("connect")` since we don't yet emit events from `ReactXnft` when individual blockchains emit the `connect` event (Idk if we should since as we add more chains this becomes one more place to make changes to). 

Ideally `window.xnft.solana.publicKey` should be defined when `connect` is emitted on `window.xnft`, but there could be a race condition where it isn't and hence I've made `fetchDegodTokens` run on when `window.xnft.solana` emits `connect`